functions/src/index.ts

```diff
@@ -34,12 +30,14 @@ exports.thingspeakToFirestore = functions
   .runWith(thingspeakToFirestoreRuntimeOpts)
   .pubsub.schedule('every 2 minutes')
   .onRun(async () => {
-    const sensorList = (await firestore.collection('/sensors').get()).docs;
+    const sensorDocQuerySnapshot = await firestore
```

Comment:
Is there a better name for this? It feels a little unclear, especially with all the extra stuff at the end and it obscuring an `isActive` requirement